stl/inc/format

```diff
@@ -756,6 +768,18 @@ public:
     }
 };
 
+// Fetch the value of an argument associated with a dynamic
+// width or precision specifier. This will be called with either
+// _Width_checker or _Precision_checker as "_Handler".
+template <class _Handler, class _FormatArg>
+static constexpr int _Get_dynamic_specs(_FormatArg _Arg) {
+    unsigned long long _Val = _STD visit_format_arg(_Handler(), _Arg);

```

Comment:
I believe this can be const

Reply:
on that note why is this internally linked.